lib/Redis/ScriptCache.pm

```diff
@@ -1,15 +1,18 @@
 package Redis::ScriptCache;
 use strict;
 use warnings;
+use 5.10.0;
 
-our $VERSION = '0.04';
+our $VERSION = '0.05';
 
 use File::Basename;
 use File::Spec qw(
     catdir
     splitdir
 );
 use Carp;
+use Digest::SHA1 'sha1_hex';
+use Try::Tiny;
```

Comment:
I removed Try::Tiny in 22c811d. I don't like the extra dependency. And although there's a network roundtrip, performance is an acceptable objection. I hate handling the return context checking, but I don't see an easy way around it.